core/src/main/scala/kafka/controller/PartitionStateMachine.scala

```diff
@@ -252,13 +252,13 @@ class ZkPartitionStateMachine(config: KafkaConfig,
         }
       case OfflinePartition =>
         validPartitions.foreach { partition =>
-          stateChangeLog.trace(s"Changed partition $partition state from ${partitionState(partition)} to $targetState")
+          stateChangeLog.info(s"Changed partition $partition state from ${partitionState(partition)} to $targetState")
```

Comment:
What's the reasoning for keeping it in trace? Isn't it useful to know when a replica is set to offline?

When do we enter this state - is it only during topic deletion and broker failure? If so, sounds good to keep in TRACE.

I guess in the cases of broker failure it may be useful to log the total number of offline partitions in `KafkaController#onReplicasBecomeOffline`